server/auth/types/openid/helper.ts

```diff
@@ -120,3 +120,16 @@ export interface TokenResponse {
   refreshToken?: string;
   expiresIn?: number;
 }
+
+export function getExpirationDate(idToken: string | undefined) {
```

Comment:
What would you think about rewriting this as a function that takes in a token response and then extracts the expiration?  I think it would clean up that logic that was shuffled around in routes.ts and it would be easy to add a test case for it, what do you think?

Reply:
sure

Reply:
fixed